docs/performance/storage.md

```diff
@@ -47,30 +55,29 @@ CrateDB is a distributed database; nodes, shards, partitions and replicas are ti
 When a table is created, data is sharded and distributed among nodes. This
 means that the memory footprint depends on our replication and sharding strategy.
 
-Let's break down how the `775MiB` in CrateDB and the `510MiB` in PostgreSQL were
+Let's break down how the `431MiB` in CrateDB and the `510MiB` in PostgreSQL were
 obtained. For `PostgreSQL` it was straightforward:
 
 ```sql
 SELECT pg_size_pretty(pg_total_relation_size('taxi_january'));
 ```
 
-For CrateDB when a table is created, sharding and replication has to be taken into account.
-When a table is created with default values, it gets partitioned in `max(4, num_data_nodes * 2)` shards.
+In CrateDB, when a table is created, sharding and replication have to be taken into account.
+When a table is created with default values, it gets partitioned using this formula: `max(4, num_data_nodes * 2)`.
```

Comment:
Maybe elegantly refer to the sharding and scaling guideline docs at this spot?

- https://cratedb.com/docs/guide/performance/sharding.html
- https://cratedb.com/docs/guide/performance/scaling.html#number-of-shards